src/ios/Cartfile.resolved

```diff
@@ -1 +1 @@
-github "getsentry/sentry-cocoa" "6.1.4"
+github "getsentry/sentry-cocoa" "6.2.1"
```

Comment:
should we delete this file instead? seems a dup of `certfile`, maybe a similar `.lock` file?